src/components/FindWallet/WalletTable.tsx

```diff
@@ -0,0 +1,717 @@
+  },
+  {
+    label: "Connect to dapps",
+    value: "Connect to dapps",
+    filterKey: "connect_to_dapps",
+    category: "feature",
+    icon: <ConnectDapps />,
+  },
+  {
+    label: "Staking",
+    value: "Staking",
+    filterKey: "staking",
+    category: "feature",
+    icon: <Staking />,
+  },
+  {
+    label: "Swaps",
+    value: "Swaps",
+    filterKey: "swaps",
+    category: "feature",
+    icon: <Swap />,
+  },
+  {
+    label: "Layer 2",
+    value: "Layer 2",
+    filterKey: "layer_2",
+    category: "feature",
+    icon: <Layer2 />,
+  },
+  {
+    label: "Gas fee customization",
+    value: "Gas fee customization",
+    filterKey: "gas_fee_customization",
+    category: "feature",
+    icon: <GasFeeCustomization />,
+  },
+  {
+    label: "ENS support",
+    value: "ENS support",
+    filterKey: "ens_support",
+    category: "feature",
+    icon: <ENSSupport />,
+  },
+  {
+    label: "Token importing",
+    value: "Token importing",
+    filterKey: "erc_20_support",
+    category: "feature",
+    icon: <ERC20Support />,
+  },
+  {
+    label: "Buy crypto",
+    value: "Buy crypto",
+    filterKey: "buy_crypto",
+    category: "trade_and_buy",
+    icon: <BuyCrypto />,
+  },
+  {
+    label: "Withdraw crypto",
+    value: "Withdraw crypto",
+    filterKey: "withdraw_crypto",
+    category: "trade_and_buy",
+    icon: <WithdrawCrypto />,
+  },
+  {
+    label: "Multisig",
+    value: "Multisig",
+    filterKey: "multisig",
+    category: "smart_contract",
+    icon: <Multisig />,
+  },
+  {
+    label: "Social recovery",
+    value: "Social recovery",
+    filterKey: "social_recovery",
+    category: "smart_contract",
+    icon: <SocialRecover />,
+  },
+]
+
+const WalletTable = ({ data, filters, walletData }) => {
+  const [walletCardData, setWalletData] = useState(
+    walletData.map((wallet) => {
+      return { ...wallet, moreInfo: false }
+    })
+  )
+  const [firstFeatureSelect, setFirstFeatureSelect] = useState(
+    featureDropdownItems[0]
+  )
+  const [secondFeatureSelect, setSecondFeatureSelect] = useState(
+    featureDropdownItems[1]
+  )
+  const [thirdFeatureSelect, setThirdFeatureSelect] = useState(
+    featureDropdownItems[2]
+  )
+
+  const updateMoreInfo = (idx) => {
+    const temp = [...walletCardData]
+    temp[idx].moreInfo = !temp[idx].moreInfo
+    setWalletData(temp)
+  }
+
+  const filteredWallets = walletCardData.filter((wallet) => {
+    let showWallet = true
+
+    Object.keys(filters).forEach((filter) => {
+      if (filters[filter] && showWallet === true) {
+        showWallet = filters[filter] === wallet[filter]
+      }
+    })
+
+    return showWallet
+  })
+
+  const IconOption = (props) => (
+    <Option {...props}>
+      {props.data.icon}
+      {props.data.label}
+    </Option>
+  )
+
+  const IconControl = ({ children, ...props }) => (
+    <Control {...props}>
+      {props.selectProps.value.icon} {children}
+    </Control>
+  )
+
+  return (
+    <Container>
```

Comment:
To make this accessible to screen readers, we should make this a HTML `<table>` with headings, rows, and data (`<th`> `<tr>` `<td>`)

Reply:
does it have to be a table? Can it be ul > li?

Reply:
WCAG recommends using tables for any tabular data as tables read the associated column and headers. So if your keyboard is on the 'Yes' circle for MetaMask layer two, it would announce it as: 'Table row, name Metamask, column layer 2, yes'. This is much harder to do with lists.